news/2025/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer.md

```diff
@@ -0,0 +1,289 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Both as a top player and as a mapper, what are your thoughts on adding CSR to osu!catch? Would that be a bad or a good thing, and why?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>
+
+Personally, I'd love to see CSR implemented in osu!catch as well. Grinding the same maps over and over for an FC was one of the most stressful and draining parts of the game for me, and it eventually made me stop playing. If CSR were introduced to osu!catch, I believe it could motivate not just new players, but top players too.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Are you still playing competitively outside of osu!catch World Cups, or have you fully moved to mapping?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>
+
+I've mostly shifted to mapping, not just because I don't have enough time to grind, but also because I feel like I no longer stand out as a player.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+You have one specific score, [disinteGration by RINYA.G \[Collapse of the World Machine\] +HDHR](https://osu.ppy.sh/scores/1956830227), pinned in your profile. Is there any story behind this or some other reason it's pinned?
+
+![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/score.jpg)
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>
+
+I was in a Discord voice chat, randomly trying maps with mods, and somehow FC'd it first try. I thought it's pretty unique to not have my top pp score as illness lilin, so I decided to pin it.
```

Comment:
what is "illness lilin"? please link

Reply:
illness lilin is a very famous (from my memory farm) [map in osu!catch](https://osu.ppy.sh/beatmapsets/777109#fruits). I think I forgot to link it, so will do that after applying web changes.